openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -25053,10 +25113,141 @@ OS:ElectricLoadCenter:Distribution,
+       \type object-list
+       \object-list ConverterList
+  N2 , \field Maximum Storage State of Charge Fraction
+       \note Fraction of storage capacity used as upper limit for controlling charging, for all storage operation schemes.
+       \type real
+       \default 1.0
+  N3 , \field Minimum Storage State of Charge Fraction
+       \note Fraction of storage capacity used as lower limit for controlling discharging, for all storage operation schemes.
+       \type real
+       \default 0.0
+  N4 , \field Design Storage Control Charge Power
+       \note Maximum rate that electric power can be charged into storage.
+       \note Storage charging adjusted downward for conversion losses.
+       \note Rate is modified by fractional values in the schedule named in the next field. 
+       \note Required field when using Storage Operation Schemes FacilityDemandLeveling or TrackChargeDischargeSchedules.
+       \type real
+       \units W
+  A14, \field Storage Charge Power Fraction Schedule Name
+       \note Controls timing and magnitude of charging storage.
+       \note Required field if Storage Operation Scheme is set to TrackChargeDischargeSchedules.
+       \note Schedule values should be fractions from 0.0 to 1.0, inclusive.
+       \type object-list
+       \object-list ScheduleNames
+  N5 , \field Design Storage Control Discharge Power
+       \note Maximum rate that electric power can be discharged from storage.
+       \note Rate is modified by fractional values in the schedule named in the next field. 
+       \note Required field when using Storage Operation Schemes FacilityDemandLeveling or TrackChargeDischargeSchedules.
+       \type real
+       \units W
+  A15, \field Storage Discharge Power Fraction Schedule Name
+       \note Controls timing and magnitude of discharging storage
+       \note Required field if Storage Operation Scheme is set to TrackChargeDischargeSchedules.
+       \note Schedule values should be fractions from 0.0 to 1.0, inclusive.
+       \type object-list
+       \object-list ScheduleNames
+  N6,  \field Storage Control Utility Demand Target
+       \note Target utility service demand power for discharge control.  Storage draws are adjusted upwards for conversion losses. 
+       \note Required field for FacilityDemandLeveling storage operation scheme
+       \type real
+       \units W
+  A16; \field Storage Control Utility Demand Target Fraction Schedule Name
+       \note Modifies the target utility service demand power over time.
+       \note Schedule values should be fractions from -1.0 to 1.0, inclusive.
+       \note if omitted a schedule value of 1.0 is used. Negative values indicate export to grid
+       \note Schedule is used if Storage Operation Scheme is set to FacilityDemandLeveling.
+       \type object-list
+       \object-list ScheduleNames
+
+OS:ElectricLoadCenter:Storage:Converter,
```

Comment:
I' just starting out on this one. I implemented storage, expanded the elcd, that good.
But realized I also need the converter so I just added it to the idd but I've got everything else to do on this one

Envoyé de mon iPhone

> Le 25 juin 2016 à 00:06, Dan Macumber <notifications@github.com> a écrit :
> 
> In openstudiocore/resources/model/OpenStudio.idd:
> 
> > +       \type object-list
> > +       \object-list ScheduleNames
> > +  N6,  \field Storage Control Utility Demand Target
> > +       \note Target utility service demand power for discharge control.  Storage draws are adjusted upwards for conversion losses. 
> > +       \note Required field for FacilityDemandLeveling storage operation scheme
> > +       \type real
> > +       \units W
> > +  A16; \field Storage Control Utility Demand Target Fraction Schedule Name
> > +       \note Modifies the target utility service demand power over time.
> > +       \note Schedule values should be fractions from -1.0 to 1.0, inclusive.
> > +       \note if omitted a schedule value of 1.0 is used. Negative values indicate export to grid
> > +       \note Schedule is used if Storage Operation Scheme is set to FacilityDemandLeveling.
> > +       \type object-list
> > +       \object-list ScheduleNames
> > +
> > +OS:ElectricLoadCenter:Storage:Converter,
> @jmarrec does this object get translated?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>